extensions/amp-story/1.0/pagination-buttons.js

```diff
@@ -246,6 +246,19 @@ export class PaginationButtons {
       }
     );
 
+    this.storeService_.subscribe(StateProperty.PAGE_IDS, () => {
+      // Make sure the story is laid out before updating.
+      this.forwardButton_.element.addEventListener(
+        EventType.STORY_LOADED,
```

Comment:
Does this work? I thought the `STORY_LOADED` event was dispatched only once, and on the `amp-story` element

Reply:
It does work. And I need this because sometimes it was calling `onCurrentPageIndexUpdate_` before the story was loaded and it was throwing some errors related to the bookend.

Reply:
I'm probably missing something here, could you please explain how this works?
The two things I don't understand are: 
- the `EventType.STORY_LOADED` event is never triggered on `this.forwardButton_.element`
- the `EventType.STORY_LOADED` event is only triggered once when rendering the story, at the very beginning. You're appending the `amp-live-list` after this event is dispatched, and it is never triggered again: how can this callback get executed?